src/server/qgsserverrequest.cpp

```diff
@@ -40,7 +40,7 @@ QgsServerRequest::~QgsServerRequest()
 
 }
 
-QString QgsServerRequest::getHeader( const QString& name ) const
+QString QgsServerRequest::getHeader( const QString& /*name*/ ) const
 {
   return "";
```

Comment:
Not sure about this place, but in 90% I use an invalid `QString()` instead of an empty `""`, but really depends on the usecase.